scripts/run_lighthouse_accessibility_tests.py

```diff
@@ -0,0 +1,188 @@
+        p.kill()
+
+    for p in processes_to_kill:
+        common.kill_processes_based_on_regex(p)
+
+    common.stop_redis_server()
+
+
+def run_lighthouse_puppeteer_script():
+    """Runs puppeteer script to collect dynamic urls."""
+    puppeteer_path = os.path.join(
+        'core', 'tests', 'puppeteer', 'lighthouse_setup.js')
+    bash_command = [common.NODE_BIN_PATH, puppeteer_path]
+
+    try:
+        script_output = subprocess.check_output(bash_command).split('\n')
+        python_utils.PRINT(script_output)
+        for url in script_output:
+            export_url(url)
+        python_utils.PRINT(
+            'Puppeteer script completed successfully.')
+
+    except subprocess.CalledProcessError:
+        python_utils.PRINT(
+            'Puppeteer script failed. More details can be found above.')
+        sys.exit(1)
+
+
+def export_url(url):
+    """Exports the url to an environmental variable."""
+    url_list = url.split('/')
+    if 'collection_editor' in url:
+        os.environ['collection_editor'] = url_list[5]
+    elif 'create' in url:
+        os.environ['exploration_editor'] = url_list[4]
+    elif 'topic_editor' in url:
+        os.environ['topic_editor'] = url_list[4]
+    elif 'story_editor' in url:
+        os.environ['story_editor'] = url_list[4]
+    elif 'skill_editor' in url:
+        os.environ['skill_editor'] = url_list[4]
+
+
+def run_webpack_compilation(source_maps=False):
+    """Runs webpack compilation."""
+    max_tries = 5
+    webpack_bundles_dir_name = 'webpack_bundles'
+    for _ in python_utils.RANGE(max_tries):
+        try:
+            webpack_config_file = (
+                build.WEBPACK_DEV_SOURCE_MAPS_CONFIG if source_maps
+                else build.WEBPACK_DEV_CONFIG)
+            subprocess.check_call([
+                common.NODE_BIN_PATH, WEBPACK_BIN_PATH, '--config',
+                webpack_config_file])
+        except subprocess.CalledProcessError as error:
+            python_utils.PRINT(error.output)
+            sys.exit(error.returncode)
+            return
+        if os.path.isdir(webpack_bundles_dir_name):
+            break
+    if not os.path.isdir(webpack_bundles_dir_name):
+        python_utils.PRINT(
+            'Failed to complete webpack compilation, exiting ...')
+        sys.exit(1)
+
+
+def run_lighthouse_checks():
+    """Runs the lighthouse accessibility checks."""
+
+    lhci_path = os.path.join('node_modules', '@lhci', 'cli', 'src', 'cli.js')
+    bash_command = [
+        common.NODE_BIN_PATH, lhci_path, 'autorun',
+        '--config=.lighthouserc-a11y.js']
+
+    try:
+        subprocess.check_call(bash_command)
+        python_utils.PRINT(
+            'Lighthouse accessibility checks completed successfully.')
+    except subprocess.CalledProcessError:
+        python_utils.PRINT(
+            'Lighthouse accessibility checks failed.' +
+            'More details can be found above.')
+        sys.exit(1)
+
+
+def enable_webpages():
```

Comment:
Most of the function in this file looks like a duplicate code from `run_lighthouse_tests.py`. Maybe have an issue to remove this duplicate codes and have a separate PR for the same!

Reply:
Please don't leave duplicate code in the codebase. Maybe worth combining the scripts instead.